central/scanner/handler.go

```diff
@@ -93,6 +107,16 @@ func generateFilesForScannerV1(params *apiparams.Scanner, clusterType storage.Cl
 			"scanner-db-cert.pem": scannerDBCert.CertPEM,
 			"scanner-db-key.pem":  scannerDBCert.KeyPEM,
 			"scanner-db-password": dbPassword,
+
```

Comment:
This comment is about `CommonConfig`. There are scannerV4 configs in image flavor. You might need to pass those scannerV4 image tags as part of `CommonConfig`

Reply:
will look into that, thank you!

Reply:
Added a ticket here: https://issues.redhat.com/browse/ROX-22036